packages/react-native/src/utils/error/ErrorReporter.ts

```diff
@@ -26,15 +25,15 @@ class ErrorReporter {
 
   constructor(platform: string) {
     this.noticeUrl = 'https://api.honeybadger.io/v1/notices'
-    this.apiKey = ErrorReporterConfig.honeybadger_api_key
+    this.apiKey = process.env.HONEYBADGER_API_KEY_REACT_NATIVE || 'undefined'
```

Comment:
The string `'undefined'` and not the value `undefined`?

Reply:
`apiKey` must be a string. Changing it to `process.env.HONEYBADGER_API_KEY_REACT_NATIVE || ''`